plugins/modules/ipadnsconfig.py

```diff
@@ -64,14 +70,19 @@
     description: |
       Work on dnsconfig or member level. It can be one of `member` or
       `dnsconfig`. Only `forwarders` can be managed with `action: member`.
+    type: str
     default: "dnsconfig"
     choices: ["member", "dnsconfig"]
   state:
```

Comment:
Does `state` and `action` have a explicit `required: False`? I've seen it in other modules.